rpc_jobs/lem_multi_node_aio.yml

```diff
@@ -59,19 +59,22 @@
           TESTR_OPTS: ""
       - string:
           name: STAGES
-          default: "Connect Slave, Prepare Multi-Node AIO, Prepare RPC Configs, Deploy RPC w/ Script, Destroy Slave"
+          default: "Connect Slave, Prepare LEM AIO, Prepare Multi-Node AIO, Prepare RPC Configs, Deploy RPC w/ Script, Setup MaaS, Verify MaaS, Install Temepst, Tempest Tests, Prepare Kibana Selenium, Kibana Tests, Holland, Destroy Slave"
```

Comment:
You are right, I will remove so these can be added in https://github.com/rcbops/u-suk-dev/issues/1370.